pkg/network/tracer/tracer.go

```diff
@@ -437,8 +437,8 @@ func (t *Tracer) getConnections(buffer []network.ConnectionStats) ([]network.Con
 		return nil, 0, err
 	}
 
-	for _, conn := range active {
-		conn.IPTranslation = t.conntracker.GetTranslationForConn(conn)
+	for i, conn := range active {
+		active[i].IPTranslation = t.conntracker.GetTranslationForConn(conn)
```

Comment:
This is fixing a bug recently introduced but not yet released 😬

Reply:
Can we get rid of `conn` altogether?

Reply:
We should probably add a test around these connections get translations, if this slipped through.

Reply:
I added a regression test